test/python/fx_importer/sparse_test.py

```diff
@@ -247,8 +253,7 @@ def forward(self, x, y):
     print(m)
 
     # Run it with PyTorch torch.sparse and with TORCH-MLIR sparse_jit.
-    # TODO: run with COO, right now only CSR works
-    sparse_input = dense_input.to_sparse_csr()
+    sparse_input = dense_input.to_sparse_coo()
```

Comment:
remove L256 completely, it is already in L251
(that way, it has the same structure as the first test above)